### DISCUSSION TITLE
The package install failed

### DISCUSSION THREAD

**Discussion by dadaziqq** [jamesjiadazhan/dietaryindex#117] [Q&A] [answered]
![mmexport1672125008045.png](https://user-images.githubusercontent.com/88812790/209626633-ab738d8d-deac-4dea-8091-8c7837f82ece.png)

**Comment by jamesjiadazhan** [comment, marked as answer]
Hi, Thanks for the feedback. Could you try installing the package again? Also, please update all packages during the installation by entering 1 if the update happens, like below:  


> These packages have more recent versions available.
> It is recommended to update all of them.
> Which would you like to update?
> 
>  1: All
>

**Comment by jamesjiadazhan** [comment]
Hi dadaziqq, the dietary package has been updated. Just a friendly reminder to install the most recent version, 0.14.0.

**Comment by zhouqimark** [comment]
你好，如果你收到这封邮件。可能 只是因为我想测试一下假期自动回复功能。。
